frameworks/helloworld/src/main/dist/examples/config_update.yml

```diff
@@ -0,0 +1,90 @@
+name: {{FRAMEWORK_NAME}}
```

Comment:
I'd rather this was named "multi_task_step" (or something) since that's what's really being tested.